pkg/logs/logs.go

```diff
@@ -96,10 +84,15 @@ func Start() error {
 	atomic.StoreInt32(&isRunning, 1)
 	log.Info("logs-agent started")
 
-	// add the default sources
-	for _, source := range config.DefaultSources() {
-		sources.AddSource(source)
-	}
+	// add the default sources after the AutoConfig has been executed once because
+	// we don't want the container_collect_all to reap all containers ownership
+	go func() {
+		common.BlockUntilAutoConfigRanOnce(time.Millisecond * coreConfig.Datadog.GetDuration("ac_load_timeout"))
```

Comment:
I think `GetDuration` expects the config value to be set to a string formatted for [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration).

In general, in config, I prefer integers in a defined unit over a string duration, so that users can only set values that make sense (not `1ns`), but since this is really an internal config option I don't mind. If we go with durations let's update the default value in `config.go` to a string

Reply:
nit: no need to block and wait if there are no default sources at all